bip-musig2.mediawiki

```diff
@@ -399,6 +411,7 @@ Input:
 
 '''''PartialSigAgg(psig<sub>1..u</sub>, session_ctx)''''':
 * Let ''(Q, _, tacc<sub>v</sub>, _, _, R, e) = GetSessionValues(session_ctx)''; fail if that fails
+* Fail if ''is_infinite(R)''
```

Comment:
Is it worth adding a short comment about why we need to fail in this case?

Reply:
Done